.azure-pipelines-templates/matrix.yml

```diff
@@ -1,5 +1,5 @@
 parameters:
-  target: ["NoSGX", "SGX"]
+  target: ["NoSGX", "SGX", "SNP"]
```

Comment:
Could `NoSGX` be renamed to virtual here and elsewhere as we now have 3 variants?

Reply:
I agree, could be a separate PR?

Reply:
https://github.com/microsoft/CCF/pull/4197